.ci.yaml

```diff
@@ -461,7 +499,7 @@ targets:
       package_sharding: "--shardIndex 5 --shardCount 6"
       dependencies: >-
         [
-          {"dependency": "android_virtual_device", "version": "33"}
+          {"dependency": "android_virtual_device", "version": "34"}
         ]
 
```

Comment:
You'll need to add matching `stable` shards, or we'll lose coverage on stable since they are running the same scripts. That can be done in the same prequel PR.